src/harmony/Checkbox/Checkbox.module.css

```diff
@@ -35,6 +35,8 @@
     margin: 0;
 
     color: var(--check-mark);
+
+    vertical-align: -0.125em;
```

Comment:
magic number

Reply:
I can't explain it, however it correctly works
I saw this magic in [font-awesome docs](https://docs.fontawesome.com/web/add-icons/svg-symbols#make-sure-to-add-some-css) and tried to use it